core/jvm/src/test/scala/fs2/time/TimeSpec.scala

```diff
@@ -63,5 +63,15 @@ class TimeSpec extends AsyncFs2Spec {
         case Vector(d) => assert(d >= delay)
       }
     }
+
+    "debounce" in {
+      val delay = 100 milliseconds
+      val s1 = Stream(1, 1, 1) ++ time.sleep[IO](delay * 2) ++ Stream(2, 2) ++ time.sleep[IO](delay / 2) ++ Stream(3)
```

Comment:
To test that the last element is emitted, change to `Stream(1, 2, 3) ++ time.sleep[IO](delay * 2) ++ Stream(4, 5) ++ ... ++ Stream(6)` and assert result is `Vector(3, 6)`?